lucid/utils.py

```diff
@@ -12,6 +12,13 @@
 from typhos import TyphosDeviceDisplay, TyphosSuite
 from typhos.utils import no_device_lazy_load
 
+try:
+    from typhos.alarm import TyphosAlarmCircle
+except ImportError:
+    # TODO: likely unnecessary back-compat
+    TyphosAlarmCircle = None
```

Comment:
I think we can keep this here indefinitely to make things easier

Reply:
Noted in 20588fb